models/user/pin.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package user
+
+import (
+	"encoding/json"
```

Comment:
The linter also complained that this is the wrong import, use
```suggestion
	"code.gitea.io/gitea/modules/json"
```
instead.
Hopefully, you don't even need to change the code in the slightest.